ggml/src/ggml-backend-reg.cpp

```diff
@@ -575,7 +584,8 @@ static ggml_backend_reg_t ggml_backend_load_best(const char * name, bool silent,
         for (const auto & search_path : search_paths) {
             fs::path filename = backend_filename_prefix().native() + name_path.native() + backend_filename_extension().native();
             fs::path path = search_path / filename;
-            if (fs::exists(path)) {
+            std::error_code ec;
+            if (fs::exists(path, ec)) {
```

Comment:
This should probably log the exception if it happens so that it does not just silently fail now. 

Alternatively we could create a list of valid paths that we get from the previous iteration where they are already checked and then just use the exists function that throws an exception here.